wagtail/snippets/tests/test_snippets.py

```diff
@@ -504,8 +504,8 @@ def test_search_hello(self):
         self.assertNotIn(self.snippet_b, items)
         self.assertIn(self.snippet_c, items)
 
-    def test_search_world(self):
-        response = self.get({"q": "World"})
+    def test_search_world_autocomplete(self):
+        response = self.get({"q": "wor"})
```

Comment:
Test with a whole word as the search query still exists on the test case just above this one